index.html

```diff
@@ -236,5 +237,52 @@ <h3>Skills</h3>
         </div>
       </section>
     </main>
+    <!--Contact form begins here-->
+    <section class="contact-me--form">
+      <div class="form-bg--image"></div>
+      <div>
+        <h2 class="contact-me">Contact me</h2>
+        <div class="form-intro">
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+        <form action="https://formspree.io/f/moqrojrw" method="post">
+          <input
+            type="text"
+            name="user_name"
+            class="contact--field"
+            id="name"
+            maxlength="30"
+            required
+            placeholder="Alier"
+          /><br /><br />
+          <input
+            type="email"
+            pattern="^([a-zA-Z0-9_\-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([a-zA-Z0-9\-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$"
+            name="user_email"
+            class="contact--field"
+            id="email"
+            maxlength="30"
```

Comment:
- [ ] please try to remove the max length from  the email input which is not necessary